src/uu/install/src/install.rs

```diff
@@ -555,6 +555,22 @@ fn copy(from: &PathBuf, to: &PathBuf, b: &Behavior) -> Result<(), ()> {
         }
     }
 
+    if b.preserve_timestamps {
+        let meta = match fs::metadata(from) {
+            Ok(meta) => meta,
+            Err(f) => crash!(1, "{}", f.to_string()),
```

Comment:
I tried to keep the call to `fs::metadata` the same as the rest of the code, for example when changing the group etc. Since the error is of type `std::io::Error`, I believe it should be clear enough. Please let me know if you would prefer a more verbose message there.